io/disk/multipath_test.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Narasimhan V <sim@linux.vnet.ibm.com>
+
+"""
+Multipath Test.
+"""
+
+import os
+from pprint import pprint
+from avocado import Test
+from avocado import main
+from avocado.utils import distro
+from avocado.utils import process
+from avocado.utils import multipath
+from avocado.utils.software_manager import SoftwareManager
+
+
+class MultipathTest(Test):
+    """
+    Multipath Test functions.
+    """
+    def setUp(self):
+        """
+        Set up.
+        """
+        # Check if multipath devices are present in sytem
+        self.wwids = multipath.get_multipath_wwids()
+        if self.wwids == ['']:
+            self.skip("No Multipath Devices")
+
+        # Install needed packages
+        dist = distro.detect()
+        pkg_name = ""
+        if dist.name == 'Ubuntu':
+            pkg_name += "multipath-tools"
+        else:
+            pkg_name += "device-mapper-multipath"
+        smm = SoftwareManager()
+        if not smm.check_installed(pkg_name) and not smm.install(pkg_name):
+            self.skip("Can not install %s" % pkg_name)
+
+        # Take a backup of current config file
+        self.mpath_file = "/etc/multipath.conf"
+        if os.path.isfile(self.mpath_file):
+            cmd = "cp %s %s.bkp" % (self.mpath_file, self.mpath_file)
+            process.system(cmd)
```

Comment:
cp command not looks good can you please use `shutil` utility 

code can be 

`shutil.copyfile('/etc/multipath.conf','/etc/multipath.conf.bkp')`